src/doc/DocAnnotator.js

```diff
@@ -364,6 +364,10 @@ class DocAnnotator extends Annotator {
     unbindDOMListeners() {
         super.unbindDOMListeners();
 
+        if (!this.container || !this.annotatedElement) {
```

Comment:
Is it possible to have one but not the other? If so, the one that still exists won't have its handlers removed.